docs/authorization.md

```diff
@@ -230,6 +230,41 @@ i.e. a claimed resource is bound to the same maximal permission policy. Only the
 
 TBD: Example
 
+## Consumer Permission Claims Authorizer
+
+The requested resource type might be a claimed resource of a PermissionClaim in an APIExport
+referenced by an APIBinding in the workspace being accessed.
+In this case access to the requested resource is constrained by the field `consumerVerbs` of the underlying permission claim.
+
+If the requested resource type is claimed by different API Exports in the same workspace,
+the intersection of all verbs of all APIExports is being granted to the user.
```

Comment:
user = consumer?

Reply:
yes, i shouldn't have used user 👍 I will just write "... is being granted.".